specification/ai/data-plane/Face/preview/v1.2-preview.1/Face.json

```diff
@@ -6981,25 +6981,6 @@
         "livenessOperationMode"
       ]
     },
-    "CreateLivenessWithVerifySessionMultipartContent": {
```

Comment:
This one I htink is correct, this model is now not used anywhere in the swagger due to params being inline